.circleci/config.yml

```diff
@@ -360,13 +385,14 @@ jobs:
               --version circle-ci-test \
               --key-file circle-sa-key.json \
               --promote
-      - manage_api_cache:
-          save: true
```

Comment:
gradle cache saving will never happen in `*-deploy-to-*` jobs because they requires other jobs that will save gradle cache if cache does not exists.

Reply:
are you saying one job overrides the other's cache? That seems odd.

Reply:
no, not saying about cache override. 
cache save only happens once if none exists and it happens on whatever the job that finishes first. deploy-to-* jobs requires other jobs run and pass first, so cache save will always be certain not happening here.